src/stockfish.ts

```diff
@@ -65,6 +65,8 @@ export class StockfishPlugin {
     if (this.isVariant()) {
       if (this.variant === 'threeCheck')
         return this.setOption('UCI_Variant', '3check')
+      else if (this.variant === 'antichess')
+        return this.setOption('UCI_Variant', 'giveaway')
```

Comment:
@shabbychef looking at the lichess website's Stockfish protocol, I see stockfish.js (which is used only by the web plugin in this repository!) **must** [be passed `giveaway`](https://github.com/ornicar/lila/commit/c4be12ee3afc8332c4f4093d2c2fb86696d7fc98) instead of `antichess`. I believe the native implementations provided by capacitor-stockfish / capacitor-stockfish-variants use https://github.com/ddugovic/Stockfish. After digging around that code a little, it appears "giveaway" is a subvariant of "anti," though I can't find any documentation as to how the two differ.

Anyway, long story short, I'd probably keep the "antichess" string in general and only patch the web behavior, e.g.:
```suggestion
      else if (Capacitor.platform === 'web' && this.variant === 'antichess')
        return this.setOption('UCI_Variant', 'giveaway')
```

Reply:
got it, I did not understand how the app was working at all. I am testing my code in the browser, not in android studio, as the latter is too hard.
Antichess, Suicide and Giveaway differ in how the winner is determined when the active player can not make a move (stuck pawns), which can result in slight strategic differences. In the responses from the (apparently web only) stockfish I saw only the `UCI_variant`s of `giveaway`, `atomic`, `crazyhouse`, `horde`, `kingofthehill`, `racingkings` and `threecheck`. This also means that passing `3check` for the web platform is indeed broken.
To separate the issues, I will pull this logic out of this PR and put into another.

Reply:
OK, peeled that out and put into PR #1833.